Unreal/SystemLocalized/det/UnrealI.det

```diff
@@ -52,13 +52,13 @@ MenuName="Behemoth"
 ; EN: NameArticle=" a "
 NameArticle=" ein "
 ; EN: MenuName="Blob"
-MenuName="Klecks"
+MenuName="Schleim"
```

Comment:
I'd suggest to keep it as "Blob" rather than "Schleim". For detailed reason see Line 61.

Reply:
```suggestion
MenuName="Blob"
```